### PR TITLE
adb from registry avail,docs:jq,sqladmin

### PR DESCRIPTION
**Pull request by dariustehrani**
* azure databricks provider now pointing to terraform registry. (old instructions were broken) + documented in readme, added /bin folder accordingly 
* jq dependency and sqladmin described README.

### CODE REVIEW

File: terraform.tfvars [hunk 1]
```diff
@@ -1,6 +1,7 @@
-region                     = "eastus2"
-data_lake_name             = "tfadl"
+region                     = "westeurope"
```
Comment (sdebruyn) [inline terraform.tfvars:1]: Please undo these changes

File: README.md [hunk 2]
```diff
@@ -75,6 +71,14 @@ The module uses some workarounds for features that are not yet available in the
 
 The module uses some workarounds for features that are not yet available in the Azure providers. Therefore, you need to have [PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell) installed.
 
+### jq
+The module uses jq to extract Databricks parameters during the deployment. Therefore, you need to haved [jq](https://stedolan.github.io/jq/download/) installed.
+
+### Setting Admin Username and password
```
Comment (sdebruyn) [inline README.md:77]: Thanks for the docs about jq, the other variables are listed on the Terraform module page. I'd like to keep them in one place so that it's easier to manage.

File: README.md [hunk 3]
```diff
@@ -52,11 +52,7 @@ This module works on macOS and Linux.
 
 ### Databricks provider installation
 
-The module is using the [Databricks Terraform provider](https://github.com/databrickslabs/terraform-provider-databricks). This provider is not in the registry yet and would have to be installed manually. This can be done with the command below:
-
-```sh
-curl https://raw.githubusercontent.com/databrickslabs/terraform-provider-databricks/master/godownloader-databricks-provider.sh | bash -s -- -b $HOME/.terraform.d/plugins
-```
+The module is using the [Databricks Terraform provider](https://github.com/databrickslabs/terraform-provider-databricks). This provider is in available from the Terraform registry.
```
Comment (sdebruyn) [inline README.md:55]: Typo: is in

File: terraform.tfvars (hunk 1 above)
Comment (dariustehrani) [reply terraform.tfvars]: fixed.

File: README.md (hunk 2 above)
Comment (dariustehrani) [reply README.md]: removed the  the hint on user credentials.

File: README.md (hunk 3 above)
Comment (dariustehrani) [reply README.md]: fixed the typo. A version bump to TF 0.13 is necessary to use the module from the tf registry. therefore I have changed the terraform.yml to use 0.13.0 in the testpipeline.
